src/main/java/peoplesoft/logic/commands/person/PersonDeleteCommand.java

```diff
@@ -46,7 +46,7 @@ public CommandResult execute(Model model) throws CommandException {
         model.deletePerson(p);
         // Deletes employment associations
         Employment.getInstance().deletePerson(p);
-        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, p));
+        return new CommandResult(String.format(MESSAGE_DELETE_PERSON_SUCCESS, p.getName()));
```

Comment:
should we also have the index here to disambiguate between multiple `Person`s with the same name?

(are we able to get the index here?)

Reply:
Too much effort to get the index. Besides, the user would have typed the index in themselves a few moments ago.